src/apiCalls.js

```diff
@@ -0,0 +1,10 @@
+import endpoints from './endpoints.js';
+
+export const postNewEmailSubscriber = (newEmailSubscriber) => {
+  return fetch(endpoints.submit, {
```

Comment:
As I mentioned in the `eb-api-post` PR, what do you think about renaming `endpoints.submit` to `endpoints.users`? I think it'd be super helpful to know which endpoint we're actually hitting since that information is being held in the `endpoints.js` file.